ddtrace/api.py

```diff
@@ -106,9 +122,19 @@ class API(object):
     # This ought to be enough as the agent is local
     TIMEOUT = 2
 
-    def __init__(self, hostname, port, headers=None, encoder=None, priority_sampling=False):
+    def __init__(self, hostname, port, path=None, headers=None, encoder=None, priority_sampling=False):
```

Comment:
Should we be more "verbose" with `path`? e.g. `unix_path`/`uds_path`, or something that signifies a unix socket path instead of a URL path?

Reply:
That's a good idea. I'll fix it.